static/app/components/replaysOnboarding/utils.tsx

```diff
@@ -17,16 +15,6 @@ export function isPlatformSupported(platform: undefined | PlatformIntegration) {
   return platform?.id ? replayPlatforms.includes(platform?.id) : false;
 }
 
-export function splitProjectsByReplaySupport(projects: Project[]) {
```

Comment:
don't need this function anymore -- we do the same functionality within the hook